packages/pds/src/services/account/index.ts

```diff
@@ -588,6 +595,15 @@ export class AccountService {
       await this.db.db.insertInto('user_pref').values(putPrefs).execute()
     }
   }
+
+  // @TODO cache w/ in-mem lookup
+  async getPds(pdsDid: string) {
```

Comment:
okay this is inspiring me to actually use the pdsDid in the mod refactor i did 😛